src/app/app.component.ts

```diff
@@ -7,5 +8,34 @@ import { Component } from '@angular/core';
     standalone: false
 })
 export class AppComponent {
-  title = 'cv';
-}
+    isSidenavOpen = false;
+    activeTabIndex = 0; 
+    darkTheme: boolean = false;
+  
+    constructor(private isDarkTheme: ThemeService) {}
+
+    ngOnInit(): void {
```

Comment:
for ngOnInit no need to add return type `: void` 
as ngOnInit never returns anythintg